src/SignalR/common/Http.Connections/src/Internal/HttpConnectionContext.cs

```diff
@@ -542,7 +542,7 @@ private async Task ExecuteApplication(ConnectionDelegate connectionDelegate)
 
             // Jump onto the thread pool thread so blocking user code doesn't block the setup of the
             // connection and transport
-            await AwaitableThreadPool.Yield();
+            await Task.Yield();
```

Comment:
This should be equivalent in the default case as there's no sync context or custom task scheduler.